traffic_ops/traffic_ops_golang/cachegroup/cachegroups.go

```diff
@@ -687,18 +699,25 @@ func (cg *TOCacheGroup) handleCoordinateUpdate() (*int, error, error, int) {
 		return nil, nil, nil, http.StatusOK
 	}
 
-	if err = cg.updateCoordinate(); err != nil {
+	err = cg.updateCoordinate()
+	if err != nil {
+		if err == ErrDuplicateExist {
```

Comment:
To compare errors, [`errors.Is`](https://pkg.go.dev/errors#Is) should be used.

Reply:
updated.